docs/local_install_docker_compose.md

```diff
@@ -1,102 +1,139 @@
-# Local install of Substra (macOS; UNIX like system)
-
-This guide will help you to get Substra source code and to start a local instance with 2 organisations: owkin and chu-nantes.
-
-## Dependencies
-
-- docker: [Docker Desktop](https://www.docker.com/products/docker-desktop)
-- python 3 (recommended 3.6 or 3.7)
-  - it is recommended to use a virtual environment to install Substra packages (for instance [virtualenv](https://virtualenv.pypa.io/en/latest/))
-- yarn
-- redis server
-
-## Configuration
-
-- docker:
-  - on macOS, by default, docker has access only to the user directory.
+# Local install of Substra (macOS & Ubuntu/Debian)
+
+- [Local install of Substra (macOS & Ubuntu/Debian)](#local-install-of-substra-macos--ubuntudebian)
+  - [1. Dependencies](#1-dependencies)
+  - [2. Configuration](#2-configuration)
+    - [Docker](#docker)
+    - [Network](#network)
+  - [3. Get source code](#3-get-source-code)
+  - [4. Install command line interface](#4-install-command-line-interface)
+  - [5. Start Substra](#5-start-substra)
+    - [Start the Substra network](#start-the-substra-network)
+    - [Start the Django backend](#start-the-django-backend)
+    - [Start frontend](#start-frontend)
+  - [6. Add some assets to Substra](#6-add-some-assets-to-substra)
+  - [7. Useful aliases](#7-useful-aliases)
+
+This guide will help you to get Substra source code and to start a local instance with two nodes: `owkin` and `chunantes`.
+
+## 1. Dependencies
+
+- Docker: 
+  - Mac: [Docker Desktop](https://www.docker.com/products/docker-desktop)
+  - Ubuntu/Debian: `sudo apt install docker docker-compose`
+- Python 3 (recommended 3.6 or 3.7)
+  - It is recommended to use a virtual environment to install Substra packages (for instance [virtualenv](https://virtualenv.pypa.io/en/latest/))
+- [Yarn](https://yarnpkg.com/getting-started/install)
+- Redis Server:
+  - [Download page](https://redis.io/download) 
+  - Or just: `sudo apt install redis-server`
+
+## 2. Configuration
+
+### Docker
+
+  - On macOS, by default, docker has access only to the user directory.
     - Substra requires access to a local folder that you can set through the `SUBSTRA_PATH` variable env (defaults to `/tmp/substra`). Make sure the directory of your choice is accessible by updating accordingly the docker desktop configuration (`Preferences` > `File Sharing`).
-    - Ensure also that the docker daemon has enough resources to execute the ML pipeline, for instance: CPUs>1, Memory>4.0 GiB (`Preferences` > `Advanced`).
-  - on Linux environment, please refer to this [guide](https://github.com/SubstraFoundation/substra-backend/blob/master/README.md) to configure docker.
+    - Also ensure that the docker daemon has enough resources to execute the ML pipeline, for instance: CPUs>1, Memory>4.0 GiB (`Preferences` > `Advanced`).
+  - On Linux environment, please refer to this [guide](https://github.com/SubstraFoundation/substra-backend/blob/master/README.md) to configure docker.
```

Comment:
Isn't it this link https://github.com/SubstraFoundation/substra-backend/blob/master/doc/linux-userns-guide.md ?

Reply:
Oh thank you!